src/service/api.tsx

```diff
@@ -134,7 +138,47 @@ const api = {
     return obj.data.results;
   },
 
-  changeImgLinks(url: string, size: string) {
+  async getSearchFileredList(dataFilter:
```

Comment:
Fix function name

Reply:
fixed 
![image](https://user-images.githubusercontent.com/41084902/115217228-f57a1e80-a10d-11eb-858e-b41c8c014437.png)

Reply:
Cannot see the fix. If this PR introduces change it should be fixed here....